Design/Rule0023AlwaysSpecifyFieldgroups.cs

```diff
@@ -20,6 +21,10 @@ private void CheckFieldgroups(SymbolAnalysisContext ctx)
             try
             {
                 ITableTypeSymbol table = (ITableTypeSymbol)ctx.Symbol;
+                if (table.Length > 0)
+                {
+                    if (IsTableOfTypeSetupTable(table.Fields[0])) return;
```

Comment:
I would consider to pass in the whole table here...

Reply:
Good idea, done.